Cod3rsGrowth.Web/wwwroot/webapp/controller/BaseController.js

```diff
@@ -0,0 +1,18 @@
+sap.ui.define([
+	"sap/ui/core/mvc/Controller",
+	"sap/ui/core/UIComponent"
+], function(Controller, UIComponent) {
+	"use strict";
+
+	return Controller.extend("coders-growth.controller.BaseController", {
+		obterRotiador : function () {
+			return UIComponent.getRouterFor(this);
+		},
+
+
+		navegarPara: function (rotaDestino) {
+			if (rotaDestino) { this.obterRotiador().navTo(rotaDestino); }
+			else { this.obterRotiador().navTo("home"); }
```

Comment:
colocar home em constante, sempre colocar as strings em constantes. mesmo esquema do back end, dependendo da quantidade de usos, ela vira global. se nao, escopo local.
se for global, tudo em maiuscula, separado por _
se for local, camelCase